client/src/containers/AssetInfo/view.jsx

```diff
@@ -18,7 +18,7 @@ class AssetInfo extends React.Component {
   render () {
     const { editable, asset } = this.props;
     const { claimViews, claimData } = asset;
-    const { channelName, claimId, channelShortId, description, name, fileExt, contentType, host, certificateId } = claimData;
+    const { channelName, claimId, channelShortId, description, name, fileExt, contentType, host, certificateId, license, licenseUrl, transactionTime } = claimData;
```

Comment:
If you break this line into multiple lines, will the linter allow it?  If not, please ping me